src/main/kotlin/com/zepben/evolve/cim/iec61968/infiec61968/infassetinfo/RelayInfo.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61968.infiec61968.infassetinfo
+
+import com.zepben.evolve.cim.iec61968.assets.AssetInfo
+import com.zepben.evolve.services.common.extensions.asUnmodifiable
+import com.zepben.evolve.services.common.extensions.typeNameAndMRID
+import java.util.function.BiConsumer
+
+/**
+ * Relay Datasheet Information.
+ *
+ * @property curveSetting The type of curve used for the Relay.
+ */
+class RelayInfo(mRID: String = "") : AssetInfo(mRID) {
+
+    var curveSetting: String? = null
+    private var _recloseDelays: MutableList<Double>? = null
```

Comment:
Was there any reason for this to be public beforehand?

Reply:
nah should have been private